### DISCUSSION TITLE
Weather window size

### DISCUSSION THREAD

**Discussion by ghost** [valinet/ExplorerPatcher#741] [General]
Hey, I tried that new weather option, but have some problems with the size of the detailed screen when you click on it. I don't know if my resolution has anything to do with it (I have 3840 x 2160). Is it possible to set it up without scroll bars? Thanks.

<img width="494" alt="weather" src="https://user-images.githubusercontent.com/97186640/151434638-8ca6ddc5-aaa6-48db-91d5-72c31dbd1a34.png">

**Comment by valinet** [comment]
Hi

Yeah, it's a bug, I noticed it today as well, and have already fixed it, once I review the code and make commits, I will push them as a new release. Hopefully not more than 1 hour from now.

Thanks for the report.
